e2e/test/scenarios/dashboard/tabs.cy.spec.js

```diff
@@ -82,9 +89,86 @@ describe("scenarios > dashboard > tabs", () => {
   });
 
   it("should allow moving dashcards between tabs", () => {
-    // TODO
+    visitDashboardAndCreateTab({
+      dashboardId: ORDERS_DASHBOARD_ID,
+      save: false,
+    });
+
+    cy.findByRole("tab", { name: "Tab 1" }).click();
+
+    cy.log("should stay on the same tab");
+    cy.findByRole("tab", { selected: true }).should("have.text", "Tab 1");
+
+    const originalSize = {};
+    getDashboardCard(0).then(element => {
+      originalSize.width = element.outerWidth();
```

Comment:
setting values from a `.then` looks very ugly to me but it's what's suggested by cypress: https://docs.cypress.io/api/commands/should#Compare-text-values-of-two-elements

Reply:
Could you use `cy.wrap(...).as(alias);` to save the size and `cy.get("@alias")` to get it back later? We use this approach for storing values in tests

Reply:
Thanks, this looks better than